src/EnergyPlus/SingleDuct.cc

```diff
@@ -4883,8 +4921,41 @@ namespace SingleDuct {
 		if ( BeginDayFlag ) {
 		}
 
-		if ( FirstHVACIteration ) {
+		if ( MyEnvrnFlag( ATMixerNum ) ) {
+			if ( ZoneEquipConfig( SysATMixer( ATMixerNum ).ZoneEqNum ).AirLoopNum > 0 ) {
+				SysATMixer( ATMixerNum ).AirLoopNum = ZoneEquipConfig( SysATMixer( ATMixerNum ).ZoneEqNum ).AirLoopNum;
+				MyEnvrnFlag( ATMixerNum ) = false;
+			}
+		}
+
+		//TRANE BEG: DCV
+		Real64 mDotFromOARequirement( 0.0 );
+		if ( !SysATMixer( ATMixerNum ).NoOAFlowInputFromUser ) {
+			Real64 airLoopOAFrac( 0.0 );
+			if ( SysATMixer( ATMixerNum ).AirLoopNum > 0 ) {
+				airLoopOAFrac = DataAirLoop::AirLoopFlow( SysATMixer( ATMixerNum ).AirLoopNum ).OAFrac;
+				if ( airLoopOAFrac > 0.0 ) {
+					Real64 vDotOAReq = CalcDesignSpecificationOutdoorAir( SysATMixer( ATMixerNum ).OARequirementsPtr, SysATMixer( ATMixerNum ).ZoneNum, true, true );
+					Real64 rhoAir = PsyRhoAirFnPbTdbW( Node( SysATMixer( ATMixerNum ).PriInNode ).Press, Node( SysATMixer( ATMixerNum ).PriInNode ).Temp, Node( SysATMixer( ATMixerNum ).PriInNode ).HumRat );
+					mDotFromOARequirement = vDotOAReq * rhoAir / airLoopOAFrac;
```

Comment:
This should be StdRhoAir, no?  Most places that call CalcDesingSpecificationOutdoorAir apply StdRhoAir, but only one does not.  That maybe should be fixed.
StdRhoAir - DualDuct, PurchasedAir, SingleDuct VAV, ZoneEquipmentManager (sizing)
local density - FanCoil